lib/db.ts

```diff
@@ -481,6 +481,53 @@ export async function updateUserConfig(
   return newConfig
 }
 
+export async function getUsageByUserStats(
+  conn: ServerlessMysql,
+  githubUser: Session['user']
+): Promise<any> {
+  const isAdmin = await isAdminUser(conn, githubUser.login)
+  if (!isAdmin) {
+    throw new Error('Not available')
+  }
+
+  // FIXME change to admin != 1
+  const userCount: RowDataPacket[] = await conn.query(SQL`
+    select s.github_user_login as id, count(*) as count
+    from simulation_runs s, authorized_users a
+    where s.github_user_login = a.github_user_login AND a.admin = 1
+    group by s.github_user_login
+    order by s.github_user_login
+  `)
+
+  return userCount.map(row => Object.assign({}, row))
+}
+
+export async function getUsageByUserPerDayStats(
+  conn: ServerlessMysql,
+  githubUser: Session['user']
+): Promise<any> {
+  const isAdmin = await isAdminUser(conn, githubUser.login)
+  if (!isAdmin) {
+    throw new Error('Not available')
+  }
+
+  // FIXME change to admin != 1
+  const userCount: RowDataPacket[] = await conn.query(SQL`
+    select s.region_id, s.subregion_id, date_format(s.created_at, '%Y-%m-%d') as day, count(*) as count
+    from simulation_runs s, authorized_users a
+    where s.github_user_login = a.github_user_login  AND a.admin = 1
+    group by s.region_id, s.subregion_id, day
+    order by s.region_id, s.subregion_id, day;
+  `)
+
+  console.log(1111111)
```

Comment:
**Nit:** Remove this or use an actual logger?

Reply:
Shouldn't be here.